chain/src/chain.rs

```diff
@@ -179,32 +179,34 @@ impl BlockChain {
         let epoch = &self.epoch;
         let mut uncles: HashMap<HashValue, MintedUncleNumber> = HashMap::new();
         let head_block = self.head_block().block;
-        let head_number = head_block.header().number();
-        if head_number < epoch.start_block_number() || head_number >= epoch.end_block_number() {
+        let head_block_header = head_block.header().number();
+        if head_block_header < epoch.start_block_number()
+            || head_block_header >= epoch.end_block_number()
+        {
             return Err(format_err!(
                 "head block {} not in current epoch: {:?}.",
-                head_number,
+                head_block_header,
```

Comment:
if you think block_height is better than block_number, I can accept.But head_block_header maybe will think struct BlockHeader, at this is only the BlockHeader height(number)